ssi-jwk/Cargo.toml

```diff
@@ -69,6 +71,7 @@ multibase = "0.9.1"
 unsigned-varint = "0.7.1"
 num-traits = "0.2"
 num-derive = "0.3"
+ps_sig = { git = "https://github.com/alan-turing-institute/RSS.git", branch = "new_rss", optional = true }
```

Comment:
Point to a commit here? Also what to do about lots of warnings generated by the ps_sig crate (warning generated by existing ps_sig code).